jenkins/PerfCI_OpenShift_Deployment/Jenkinsfile

```diff
@@ -42,14 +42,16 @@ pipeline {
                             sudo podman rmi -f $(sudo podman images -q quay.io/ebattat/benchmark-runner 2> /dev/null)
                         fi
                         '''
-                        // Stop running containers
-                        sh 'sudo podman ps | awk \'{print $1}\' | sudo xargs -I % sudo podman stop % 2>/dev/null'
-
-                        // Remove stopped containers
-                        sh 'sudo podman ps -a | awk \'{print $1}\' | sudo xargs -I % sudo podman rm % 2>/dev/null'
-
-                        // Remove pods
-                        sh 'sudo podman pod ps | awk \'{print $1}\' | sudo xargs -I % sudo podman pod rm % 2>/dev/null'
+                        // Avoid container failure errors
+                        try {
+                            sh '''
+                                sudo podman ps | awk '{print $1}' | xargs -I % sudo podman stop % || true;
+                                sudo podman ps -a | awk '{print $1}' | xargs -I % sudo podman rm % || true;
+                                sudo podman pod ps | awk '{print $1}' | xargs -I % sudo podman pod rm % 2>&1 || true
```

Comment:
I am doing it according to Jetlag recommendation otherwise it will cause installation issue:
https://github.com/redhat-performance/jetlag/blob/main/docs/troubleshooting.md#clean-all-container-services--podman-pods

Reply:
Fair enough.  I do think you should clean up pods/containers before `podman rmi`'ing your image.

Reply:
We don't need to remove any images; we only need to remove the assisted installer pods. Otherwise, it will not install the latest OCP version

Reply:
I see the podman rmi for the benchmark-runner image above this.  But I suppose the only change here is catching exceptions, and this code has otherwise worked, so I'll approve it.